oat-sa/extension-tao-itemqti
# Feature/aut 22/add ckeditor to inline choice

https://oat-sa.atlassian.net/browse/AUT-22

related fe branch: https://github.com/oat-sa/tao-item-runner-qti-fe/tree/feature/AUT-22/add-ckeditor-to-inline-choice

**How to test**
- Create inline choice interaction
- Try to add different markup and math to the choices
- Add new / delete choices
- Show, set the response
- Save item
- Check preview

Env: http://test-anastasia.playground.kitchen.it.taocloud.org:44451/
